css/menu.css

```diff
@@ -252,3 +252,8 @@ body.disableScroll {
 [data-fl-exit-app] .fa-angle-right.linked-icon {
   display: none;
 }
+
+/* Align exit button icon with other icons */
+[data-fl-exit-app] .fl-menu-icon {
+  padding-left: 7px;
```

Comment:
@YaroslavOvdii Good for you for paying attention to the detail :+1: I actually think, however, that it looks fine without moving it by 7px. You might have felt that it's too much to the left because the icon is "heavier" on the left. However, if users start to customse each menu item with their custom icon, the exit icon will look incorrect.

So, let's not move it.

The rest of the code looks good.

**Moved back by 7px ⬇️**

![image](https://user-images.githubusercontent.com/290733/77054221-6741df00-69c7-11ea-815e-b8f8b25f9b20.png)

Reply:
@tonytlwu removed icon padding-left.